java/runtime/src/main/java/org/ray/runtime/Worker.java

```diff
@@ -1,5 +1,8 @@
 package org.ray.runtime;
 
+import com.google.common.base.Preconditions;
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
`Map` and `HashMap` do not seem to be used in your added codes.

Reply:
Good catch!